views/show.ejs

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title></title>
+</head>
+<body>
+
+<h1>Missions Show Page</h1>
+
+The <%= mission.name %> Lanches on <%= mission.launchDate %>,  Operated by <%= mission.operator %> . The type of mission will be a <%=mission.missionType%> 
+
+<img src="http://www.wallpaperbetter.com/wallpaper/15/951/740/curiosity-rover-mars-nasa-alien-landscape-robot-machine-hd-1080P-wallpaper-middle-size.jpg">
```

Comment:
instead of hard-coding in one image, try adding an image in each object in the data under "img", and have the images change accordingly on the show page